grammar/fbp.peg

```diff
@@ -17,6 +17,93 @@
 
   nodes = {};
 
+  var serialize, indexOf = [].indexOf || function(item) { for (var i = 0, l = this.length; i < l; i++) { if (i in this && this[i] === item) return i; } return -1; };
+
+  parser.serialize = function(graph) {
```

Comment:
Uhm, this code is generated from the CoffeeScript. How does it end up in here during build process?
It is not acceptable to require manual steps to keep this in sync... In general, mixing generated code and not-generated is pretty bad.